test/helpers/utils.go

```diff
@@ -475,14 +475,20 @@ func failIfContainsBadLogMsg(logs string, blacklist map[string][]string) {
 					}
 				}
 				if !ok {
-					fmt.Fprintf(CheckLogs, "⚠️  Found a %q in logs\n", fail)
-					nFailures++
+					count, _ := uniqueFailures[fail]
+					uniqueFailures[fail] = count + 1
 				}
 			}
 		}
 	}
-	if nFailures > 0 {
-		Fail(fmt.Sprintf("Found %d Cilium logs matching list of errors that must be investigated", nFailures))
+	if len(uniqueFailures) > 0 {
+		failures := make([]string, 0, len(uniqueFailures))
+		for f, c := range uniqueFailures {
+			failures = append(failures, f)
+			fmt.Fprintf(CheckLogs, "⚠️  Found %q in logs %d times\n", f, c)
```

Comment:
Bikeshedding nit: does the `⚠️` sign serve any purpose in the log msg?;-)

Reply:
It's been a while since I even thought about it but the idea is to highlight this concern in the output. It certainly gives these particular messages a certain aesthetic, breaking up the flow of the log output and helps to highlights this output (though these days there's always false positives due to certain logs triggered by our CI system :( )